spk/znc/Makefile

```diff
@@ -1,18 +1,19 @@
 SPK_NAME = znc
-SPK_VERS = 1.7.5
-SPK_REV = 13
+SPK_VERS = 1.8.1
+SPK_REV = 1
```

Comment:
Revision 14 is required.

Reply:
By the time I updated, it had to be revision 15. Done.